js/packages/web/src/views/home/auctionList.tsx

```diff
@@ -75,7 +76,25 @@ export const AuctionListView = () => {
             const id = m.auction.pubkey;
             return (
               <Link to={`/auction/${id}`} key={idx}>
-                <AuctionRenderCard key={id} auctionView={m} />
+                <AuctionRenderCard key={id} auctionView={m} onClick={() => {
+                  track('view_item', {
+                    category: 'auction',
+                    currency: "USD",
+                    value: 2 * 244, //solPrice * solValue,
+                    sol_value: 2,
+                    items: m.items.flat().map((i,j) => ({
+                      item_id: i.metadata.info.data.uri,
+                      item_name: i.metadata.info.data.name,
+                      affiliation: storefront.subdomain, // or page title?
+                      currency: 'USD',
+                      price: i.amount.toNumber() * 244, // 244 is sol price,
```

Comment:
Spot where you can bring in the metaplex sdk to get the current usd conversion. Though good to calculate this just before sending the event so the price is up to date.

https://github.com/metaplex/js